cli_reference/cli_manager/cli-manager-install.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="cli-manager-install_{context}"]
+= Installing the {cli-manager}
+include::_attributes/common-attributes.adoc[]
+:context: cli-manager-install
+
+toc::[]
+
+//Brief explanation of what the cli manager is.
+In disconnected environments it is difficult to install and manage CLI plugins. The CLI Manager was created to make installing and managing CLI plugins in disconnected environments easier.
```

Comment:
I think it is better to mention that it is Krew compatible as stated in https://github.com/openshift/cli-manager?tab=readme-ov-file#cli-manager
also this is not only bounded to disconnected environments (yes primary purpose is for disconnected). I think it is better to say both connected and disconnected and mention it will be especially beneficial for disconnected.